fuzzing/local_run.sh

```diff
@@ -0,0 +1,153 @@
+#!/bin/bash
```

Comment:
Please run shellcheck on this script.
It's a quick win and can prevent some issues like the lack of double quoting